beacon-chain/rpc/validator/assignments.go

```diff
@@ -35,13 +35,18 @@ func (vs *Server) GetDuties(ctx context.Context, req *ethpb.DutiesRequest) (*eth
 			return nil, status.Errorf(codes.Internal, "Could not process slots up to %d: %v", epochStartSlot, err)
 		}
 	}
-
 	committeeAssignments, proposerIndexToSlot, err := helpers.CommitteeAssignments(s, req.Epoch)
```

Comment:
I think we definitely can be more efficient here, say at next epoch, since this was already calculated in previous epoch with epoch + 1 look ahead, we can skip

Reply:
Actually we might still have to run this again, since we need to return the duties to all validators in the current epoch. Although if we are running the committee cache now, it shouldnt be a problem ?